src/pdfflow/configflow.py

```diff
@@ -14,8 +14,11 @@
 # tf.config.run_functions_eagerly(True)
 
 def run_eager(flag = True):
-    """ Wraper around `run_functions_eagerly` """
-    tf.config.run_functions_eagerly(flag)
+    """ Wrapper around `run_functions_eagerly` """
+    if tf.__version__ == '2.2.0':
+        tf.config.experimental_run_functions_eagerly(flag)
+    elif tf.__version__ == '2.3.0':
```

Comment:
Umm, they have already released 2.3.1, so maybe do `>= 2.3.0`?

Reply:
Maybe there won't be major changings in the ops we use within `2.3.x` versions. So we can just modify the conditionals to be `if '2.2 in tf.__version__':

Reply:
Sure